testdata/regresstest/test_pytorch_module.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/ksh
+
+if [ $# == 0 ]; then
+  echo "Usage: ./test_pytorch_module.sh path/to/Truth/ [batched]"
+  exit 0
+fi
+
+set -u
+
+GAPC=../../../gapc
+MAKE="make"
+PYTHON="python3"
```

Comment:
It should work with just `python` as well. I think that the official interpreter binary name is still `python3`, so that should always work. Just `python` probably works in most shells as well though.